# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/openrgb/hotfix-stdint.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- OpenRGB-release_0.8/Controllers/SteelSeriesController/SteelSeriesRival3Controller.h.vanilla	2023-05-11 15:33:05.453161916 +0200
+++ OpenRGB-release_0.8/Controllers/SteelSeriesController/SteelSeriesRival3Controller.h	2023-05-11 15:33:11.464162350 +0200
@@ -8,6 +8,7 @@
 \*-----------------------------------------*/
 
 #include <string>
+#include <inttypes.h>
 #include <hidapi/hidapi.h>
 
 #include "SteelSeriesGeneric.h"
--- OpenRGB-release_0.8/Controllers/SteelSeriesController/SteelSeriesRivalController.h.vanilla	2023-05-11 15:44:48.664212691 +0200
+++ OpenRGB-release_0.8/Controllers/SteelSeriesController/SteelSeriesRivalController.h	2023-05-11 15:44:56.150213231 +0200
@@ -8,6 +8,7 @@
 \*-----------------------------------------*/
 
 #include <string>
+#include <stdint.h>
 #include <hidapi/hidapi.h>
 
 #include "SteelSeriesGeneric.h"
--- OpenRGB-release_0.8/Controllers/SteelSeriesController/SteelSeriesMouseController.h.vanilla	2023-05-11 15:44:10.292209920 +0200
+++ OpenRGB-release_0.8/Controllers/SteelSeriesController/SteelSeriesMouseController.h	2023-05-11 15:44:22.214210781 +0200
@@ -9,6 +9,7 @@
 #include <string>
 #include <hidapi/hidapi.h>
 #include <vector>
+#include <stdint.h>
 #include "SteelSeriesGeneric.h"
 
 #pragma once
--- OpenRGB-release_0.8/Controllers/ENESMBusController/ENESMBusInterface/ENESMBusInterface.h.vanilla	2023-05-11 15:45:15.683214642 +0200
+++ OpenRGB-release_0.8/Controllers/ENESMBusController/ENESMBusInterface/ENESMBusInterface.h	2023-05-11 15:45:20.861215016 +0200
@@ -7,6 +7,7 @@
 \*-----------------------------------------*/
 
 #include <string>
+#include <stdint.h>
 
 #pragma once
 
